City-of-Helsinki/drupal-hdbt
# UHF-930 Align the link arrows again

How to test:
- `composer require drupal/hdbt:dev-UHF-930-align-link-arrows-again`
- `make drush-cr`
- Go to https://helfi.docker.sh/fi/esimerkkisivu or https://hel-platform.docker.sh/fi/esimerkkisivu depending on which site you are using.
- You should now see that on the link list element the arrows on the links are no longer always in same place like in here: https://hel-platform-test.druid.fi/fi/esimerkkisivu but actually follow the link texts length (screenshot attached).
- Its worth noticing tho that if the title goes on two or more lines there could be larger space between and at least with quick look this cannot be helped.
